### PR TITLE
Rename request_tests.go to request_test.go

### PR DESCRIPTION
**Pull request by MatMaul**
This is the usual convention and it is assumed by several tools like godep.

Needed for trust support in Kubernetes : 
https://github.com/kubernetes/kubernetes/pull/32111

### CODE REVIEW

**Comment by codevulture** [comment]
+1 godeps failure reason!

**Comment by MatMaul** [comment]
Seems like the added tests were not call at all, because one is failing :) Next update should fix it.

**Comment by strigazi** [comment]
@jrperritt Can you have a look at this?

**Comment by MatMaul** [comment]
@jrperritt can we merge this please ? we are waiting for it to add trust id to Kubernetes. Thanks !

**Comment by strigazi** [comment]
@jrperritt Do you have an update? We are blocked in kubernetes side.

File: openstack/identity/v3/extensions/trust/request_test.go [hunk 1]
```diff
@@ -87,12 +66,3 @@ func TestTrustIDTokenID(t *testing.T) {
 
 	`)
 }
-
-func TestFailurePassword(t *testing.T) {
```
Comment (jrperritt) [inline openstack/identity/v3/extensions/trust/request_test.go]: Don't delete tests; fix them instead.

Comment (MatMaul) [inline openstack/identity/v3/extensions/trust/request_test.go]: This one never worked (those tests actually never ran before because of the filename), and doesn't seem really useful. I fixed the other one which was failing too. Be my guest if you understand and know how to fix this one.

Comment (jrperritt) [inline openstack/identity/v3/extensions/trust/request_test.go]: Let me try again: If you want this PR merged, fix the unit test.
